src/app/page.tsx

```diff
@@ -19,34 +15,39 @@ async function listClients() {
   return data
 }
 
-async function getCustomFields() {
-  const res = await fetch(`${copilotAPIUrl}/v1/custom-fields`, {
-    headers: {
-      'X-API-KEY': copilotAPIKey as string,
-    },
-  })
+async function getCustomFields(token: string) {
+  const res = await fetch(`${apiUrl}/api/autofill?token=${token}`)
 
   if (!res.ok) {
     throw new Error('Something went wrong while fetching client list!')
   }
 
-  const { data } = await res.json()
+  const { autofillFields } = await res.json()
+  return autofillFields
+}
+
+async function getSettings(token: string) {
+  const { data } = await fetch(`${apiUrl}/api/settings?token=${token}`).then(
```

Comment:
lets use either await  or .then and not mix them

Reply:
replaced with async/await in latest commit